compiler/x/codegen/X86Ops.ins

```diff
@@ -556,6 +556,14 @@ INSTRUCTION(CMOVG4RegMem, cmovg,
             BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_0F__, 0x4F, 0, ModRM_RM__, Immediate_0),
             PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_IntSource | IA32OpProp_TestsOverflowFlag | IA32OpProp_TestsSignFlag | IA32OpProp_TestsZeroFlag | IA32OpProp_IntTarget),
             PROPERTY1(IA32OpProp1_SourceIsMemRef)),
+INSTRUCTION(CMOVGE4RegMem, cmovge,
+            BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_0F__, 0x4d, 0, ModRM_RM__, Immediate_0),
+            PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SourceRegisterInModRM | IA32OpProp_IntSource | IA32OpProp_IntTarget | IA32OpProp_TestsOverflowFlag | IA32OpProp_TestsSignFlag | IA32OpProp_TestsOverflowFlag),
```

Comment:
`IA32OpProp_SourceRegisterInModRM` shouldn't be used because the source isn't in a register.

Reply:
Updated.